Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["zsteve <stephenz@student.unimelb.edu.au>"]
 version = "0.3.17"
 
 [deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```

Comment:
Can we just replace `eachcol` with `(view(x, :, i) for i in axes(x, 2))` (that's its definition)? Compat is a quite large package and it seems a bit strange to depend on it just for one line.

Reply:
Yep fair enough.